src/apps/persistence/app_metadata.clj

```diff
@@ -532,11 +531,6 @@
                                             {:output parameter-id})))
    (remove-workflow-map-orphans)))
 
-(defn update-app-labels
-  "Updates the labels in an app."
-  [req]
-  (relabel/update-app-labels req))
```

Comment:
This indirection was removed since `apps.persistence.app-metadata.relabel/update-app-labels` now needs to reference `apps.persistence.app-metadata/get-app-latest-version`, which was causing a circular dependency.